buddyboss/buddyboss-platform
# PROD-754 Fix/2853

### All Submissions:



### Changes proposed in this Pull Request:

Add a filter to allow for overriding the dropdown content.

Fixes #2853  .

### How to test the changes in this Pull Request:

1. Create a snippet, such as the following, to use the filter
```
add_filter ('bbp_reply_attributes_meta_box_discussion_reply_title_filter', 'modify_move_reply_dropdown', 10, 2);
function modify_move_reply_dropdown ($content, $post) {
	$content = $post->post_title;
	
return $content;
}
```

2. Open any forum and click Move in any reply's menu
3. In the next window, the dropdown for Use an existing discussion should show the topics' post_title rather than post_content

### Proof Screenshots or Video

https://user-images.githubusercontent.com/88559987/131426243-afa3e7a7-a775-4c2a-92b5-102bd2e360a6.mp4

### Other information:

*[x] Have you added an explanation of what your changes do and why you'd like us to include them?



### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

Add filter to allow users to change the content of the dropdown box when moving a reply to an existing discussion topic.
